### PR TITLE
Feature/remove useless return statements

### PR DESCRIPTION
**Pull request by babacros**
#### Checklist

- [x] Updated CHANGELOG.md.

### CODE REVIEW

File: ProjectTemplate/Services/VersionUpdate/FirebaseFetcher.swift [hunk 1]
```diff
@@ -6,7 +6,7 @@ public class FirebaseFetcher: Fetcher {
     private let remoteConfig: RemoteConfig
 
     public var version: Int? {
-        return remoteConfig.configValue(forKey: key).numberValue?.intValue
+         remoteConfig.configValue(forKey: key).numberValue?.intValue
```
Comment (LukasHromadnik) [inline ProjectTemplate/Services/VersionUpdate/FirebaseFetcher.swift]:
```suggestion
        remoteConfig.configValue(forKey: key).numberValue?.intValue
```

**Comment by olejnjak** [comment]
Removed *Podfile.lock*? Why?!

**Comment by babacros** [comment]
> Removed _Podfile.lock_? Why?!
> 
> > I'm definitely not a fan, seems less readable to me 😬 But I guess I will have to get use to it 😕
> > I have this as a topic on today's meeting 🤷‍♂

Sorry, I'm just an idiot 🙇🏻‍♂️

**Comment by LukasHromadnik** [comment]
@babacros There are conflicts in Changelog

**Pull request merged**